include/libdnf/utils/utils.hpp

```diff
@@ -132,9 +132,9 @@ inline bool GoalJobSettings::get_strict(const libdnf::ConfigMain & cfg_main) con
         case GoalSetting::SET_TRUE:
             return true;
         case GoalSetting::SET_FALSE:
-        default:
             return false;
     }
+    throw LogicError("Unknown 'best' value in GoalJobSettings");
```

Comment:
Could you replace the word "Unknown" with "Invalid"? It's a bit more specific. Also, not critical in this case, but putting the actual invalid value in the message (as a number) would be nice.

Reply:
I replaced the word "Unknown" with "Invalid".